backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -1549,9 +1562,10 @@ void createFhirBundle_TestEvent_matchesJson() throws IOException {
             StandardCharsets.UTF_8);
 
     var expectedCurrentDateTimezone =
-        new DateTimeType(date, TemporalPrecisionEnum.SECOND, TimeZone.getTimeZone("UTC"))
+        new DateTimeType(currentDate, TemporalPrecisionEnum.SECOND, TimeZone.getTimeZone("UTC"))
             .getValueAsString();
-    var expectedCurrentDateZulu = new InstantType(date).setTimeZoneZulu(true).getValueAsString();
+    var expectedCurrentDateZulu =
+        new InstantType(currentDate).setTimeZoneZulu(true).getValueAsString();
```

Comment:
If we are going to use a constant date, can we simplify this by just putting this date in the file `fhir/bundle.json` and then removing 1564 - 1568 and 1573 - 1575?